v2/trace/ddpgx/conn.go

```diff
@@ -0,0 +1,69 @@
+package ddpgx
+
+import (
+	"context"
+	"time"
+
+	"github.com/jackc/pgconn"
+	"github.com/jackc/pgtype"
+	"github.com/jackc/pgx/v4"
+	dd_ext "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+)
+
+type traceConn struct {
+	conn        Connection
+	serviceName string
+}
+
+func (o *traceConn) ConnInfo() *pgtype.ConnInfo {
+	return o.conn.ConnInfo()
+}
+
+func (o *traceConn) Begin(ctx context.Context) (pgx.Tx, error) {
+	startTime := time.Now()
+	tx, err := o.conn.Begin(ctx)
+	tryTrace(ctx, startTime, o.serviceName, "pgx", "Begin", nil, err)
```

Comment:
consider adding constants for "pgx", "Begin", etc.

Reply:
Updated